_data/simple-icons.json

```
@@ -11223,6 +11223,11 @@
             "hex": "18A5D6",
             "source": "https://ulule.frontify.com/d/EX3dK8qsXgqh/branding-guidelines"
         },
+        {
+            "title": "Umami",
+            "hex": "000000",
+            "source": "https://github.com/mikecao/umami/blob/master/assets/logo.svg"
```

Please use a permalink to the logo file as described in the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md):

> In any case, the commit hash should be part of the URL. On GitHub, you can get the correct URL by pressing `y` on the GitHub page you want to link to. You can get help at the [getting permanent links to files page](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files).

```suggestion
            "source": "https://github.com/mikecao/umami/blob/5358e28a68e0ccf4fdba95dbdc495779429f2a4f/assets/logo.svg"
```

---

Sorry! Thanks, a will take a another better look in the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md).